student.rb

```diff
@@ -15,7 +15,7 @@ class Student < Person
   # classroom - The String name of the classroom.
```

Comment:
- [ ] OPTIONAL
Kindly ensure to use `require_relative` for classes  (_line 1_) in the same source and use `require` for importing gems/ libraries instead